Sources/ParseSwift/ParseConstants.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ParseConstants.swift
+//  ParseSwift
+//
+//  Created by Corey Baker on 9/7/20.
+//  Copyright © 2020 Parse Community. All rights reserved.
+//
+
+import Foundation
+
+let kParseVersion = "0.0.1"
```

Comment:
The `k*` constants feel out of place; can these be static members on a `Constants` enum or something?

Reply:
Just added the enum. What do you think about this?